armi/materials/uranium.py

```diff
@@ -20,35 +20,280 @@
 .. [AAAFuels]  Kim, Y S, and Hofman, G L. AAA fuels handbook.. United States: N. p., 2003. Web. doi:10.2172/822554. .
 """
 
-from armi.materials.material import Material
+from numpy import interp
+
+from armi.materials.material import FuelMaterial
+from armi.nucDirectory import nuclideBases as nb
 from armi.utils.units import getTk
 
 
-class Uranium(Material):
+class Uranium(FuelMaterial):
```

Comment:
I mean, obviously, stated in this one line, this change seems reasonable.

But.... can we run downstream integration tests and simulations to show that this doesn't drastically change our baselines?  Or that we can update our baselines to match.

The downstream changes are my only high-level concern.

Reply:
Thanks!